Tribler/Test/test_my_channel.py

```diff
@@ -125,6 +129,88 @@ def do_page():
 
         self.startTest(do_page)
 
+    def test_metadata(self):
+        def take_screenshot_quit():
+            self.screenshot('Home')
+            self.quit()
+
+        def check_home():
+            self.CallConditional(20, lambda: len(self.guiUtility.frame.home.aw_panel.list.GetItems()) == 1,
+                                 take_screenshot_quit, "No thumbnail torrent in Home panel")
+
+        def on_channel_torrent_created(subject, events, object_id, data_list):
+            add_metadata = False
```

Comment:
I made a new channel manager and a rss-parser in `Tribler/Core/Modules`, which support creating multiple channels. That's why I didn't touch the old rss-feed. I will try to figure out a better way to do this.